src/Orleans/GrainDirectory/GlobalSingleInstanceRegistration.cs

```diff
@@ -11,27 +11,9 @@ namespace Orleans.GrainDirectory
     [Serializable]
     internal class GlobalSingleInstanceRegistration : MultiClusterRegistrationStrategy
     {
-        private static GlobalSingleInstanceRegistration singleton;
+        private static readonly string[] EmptyList = new string[0];
```

Comment:
I chose an empty array because the consumer (or at least some) call `.ToList()` on it, but I think it doesn't really make a difference, so I'll use Enumerable.Empty